recipes/_configure_server.rb

```diff
@@ -105,7 +105,7 @@
 
 license_group =
   case node['splunk']['node_type']
-  when :license_server, :cluster_master, :cluster_slave
+  when :cluster_master, :cluster_slave, :heavy_forwarder, :license_server
```

Comment:
Out of curiosity, with this heavy_forwarder, why do we apply the license group of Enterprise instead of Forwarder like the UF?